context/app/static/js/components/detailPage/related-entities/RelatedEntitiesTabs/RelatedEntitiesTabs.jsx

```diff
@@ -11,16 +12,24 @@ function RelatedEntitiesTabs({ entities, openIndex, setOpenIndex, ariaLabel, ren
 
   return (
     <>
-      <StyledTabs value={openIndex} onChange={handleChange} aria-label={ariaLabel}>
-        {entities.map((entity, i) => (
-          <Tab
-            label={`${entity.tabLabel} (${entity.data.length})`}
-            index={i}
-            key={entity.tabLabel}
-            data-testid={`${entity.tabLabel.toLowerCase()}-tab`}
-          />
-        ))}
-      </StyledTabs>
+      <StyledIconTab>
+        <StyledTabs value={openIndex} onChange={handleChange} aria-label={ariaLabel}>
+          {entities.map((entity, i) => (
+            <Tab
+              label={`${entity.tabLabel} (${entity.data.length})`}
+              index={i}
+              key={entity.tabLabel}
+              data-testid={`${entity.tabLabel.toLowerCase()}-tab`}
+              icon={
+                <StyledSvgIcon
+                  component={entityIconMap[`${entity.tabLabel.slice(0, entity.tabLabel.length - 1)}`]}
```

Comment:
```suggestion
                  component={entityIconMap[entity.entityType]}
```
I believe `entityType` should exist. Make sure to double check the instances for this component.

Reply:
Ah I see, I still had access to entityType...